### PR TITLE
Events Marketing Lead Form

### PR DESCRIPTION
**Pull request by stewsmith**
https://optimizely.atlassian.net/browse/MKT-752

### CODE REVIEW

File: website-guts/templates/layouts/events_form.hbs [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+- marketing_events
+---
+<div id="main">
+  <div class="header-wrapper">
+    <h1 class="page-title">{{page.title}}</h1>
+    <div class="container">
+      <div class="row">
+        <div class="columns six offset-three">
+          <p page="subtitle">{{page.subtitle}}</p>
+          <form id="events-form" method="POST" class="event-form" action="" autocomplete="off" novalidate="novalidate">
```
Comment (kylerush) [inline website-guts/templates/layouts/events_form.hbs]: @stewsmith take off the action attribute if it's not posting.

File: website-guts/templates/layouts/events_form.hbs [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+                <input id="first_name" required name="first_name" type="text" placeholder="First name">
+              </div>
+              <div class="label half right">
+                <label for="last_name">Last name</label>
+                <input id="last_name" required name="last_name" type="text" placeholder="Last name">
+              </div>
+            </div>
+            <div class="label">
+              <label for="company_name">Company name</label>
+              <input id="company_name" required class="full-width" name="company_name" type="text" placeholder="Company name">
```
Comment (kylerush) [inline website-guts/templates/layouts/events_form.hbs]: @stewsmith should be `name="company"`

File: website-guts/templates/layouts/events_form.hbs [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+            </div>
+            <div class="label">
+              <label for="email_address">Work email</label>
+              <input id="email_address" class="full-width" required name="email_address" type="text" placeholder="Work email">
```
Comment (kylerush) [inline website-guts/templates/layouts/events_form.hbs]: @stewsmith trackLead says this is supposed to be `name="email"`

**Comment by kylerush** [comment]
@stewsmith here are the names of the hidden fields (sorry i don't have values yet, need those from allison):

Lead_Source_Category__c,
Lead_Source_Subcategory__c,
Initial_Form_Source__c,
Inbound_Lead_Form_Type__c

create hidden fields for these. the values will get populate with page properties, but i don't have the values just yet.

**Comment by kylerush** [comment]
for the segment event let's do this inside of the load/success function:

```js
w.analytics.track('lead success', {
    category: 'external event',
    label: w.location.pathname
}, {
        integrations: {
           'Marketo': false
        }
});
```

be sure to use the trim trailing slash function on the label value.

**Comment by kylerush** [comment]
@stewsmith i don't have time to run `grunt test` but if you run that successfully then LGTM

**Comment by stewsmith** [comment]
@kylerush TEST PASS. Will wait to merge until we hear back about content.

**Pull request merged**